README.md

```diff
@@ -188,6 +188,87 @@ pip install 'apache-airflow[postgres,google]==3.0.0' \
 For information on installing provider distributions, check
 [providers](http://airflow.apache.org/docs/apache-airflow-providers/index.html).
 
+## Setting up Apache Airflow on Apple Silicon (Mac M1/M2/M3/M4)
```

Comment:
Hmm. Actually when looking at the whole, rendered README .... I think that should be a separate doc with link from README.md. 

Our README.md is **precious** and it should be **short** -> we already had a very, very long README and I think it's a good idea to make it **shorter** rather than **longer**. I think togetjher- this and previous chapter are long enough to be extracted to a separate `INSTALLING.md` and linking to that INSTALLING.md from README